contracts/ReversibleICO.sol

```diff
@@ -23,7 +23,7 @@ contract ReversibleICO is IERC777Recipient {
     IERC1820Registry private _erc1820 = IERC1820Registry(0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24);
     bytes32 constant private TOKENS_RECIPIENT_INTERFACE_HASH = keccak256("ERC777TokensRecipient");
 
-    IERC777 public TokenContract;
+    IERC777 public tokenContract;
```

Comment:
This i would keep uppercase as it is a class instance. Which should be differentiable in between simple value variables